### DISCUSSION TITLE
Recon_demo problem

### DISCUSSION THREAD

**Discussion by marjolein90** [spinicist/riesling#18] [Q&A] [answered]
Hi! Thanks for sharing a potentially great toolbox. I have some start up problems, hope you can help.

When running the recon_demo.ipynb, I get the following error:

![image](https://user-images.githubusercontent.com/4727288/121321894-cc585c00-c90e-11eb-8cd7-b67e15b95031.png)

I also received this error when trying to follow the example, when running:
$ bash riesling_recon.sh 

However, I could run
$ bash rrsg_recon.sh

Any help is appreciated.

Best,
Marjolein

**Comment by emilljungberg** [comment, marked as answer]
Hi Marjolein, 

Thanks for your message! We are currently in the process of updating the demo scripts. There have been some changes in how we store data in the `.h5` files and that's what is causing these issues. I'm revising the scripts right now and should have them ready for you to try out soon. I'll let you know where to find them when they are ready.

We very much welcome all input so if you have any question or problems as you continue using `riesling` please do let us know :)

**Reply by emilljungberg** [reply]
Hi again @marjolein90 

I've updated the [riesling-examples](https://github.com/spinicist/riesling-examples) repository now with a few different examples. One with a 3D Shepp Logan phantom generated with `riesling` and another using data from the ISMRM reproducibility group conjugate gradient SENSE challenge. 

Let me know how you get on with the examples and if there is anything we can improve.

**Reply by marjoleinpiek** [reply]
Same problem persists.
I also miss the file: download_data.sh

I haven't tried the new examples though, will give them a try!

**Reply by marjoleinpiek** [reply]
Same person, different account. Apologies for confusion.

**Reply by emilljungberg** [reply]
In the riesling-examples repo there is a folder called rrsg_cg-sense. The file to download the data is called `download_rrsg_data.sh`. Can you have a look at those examples and see if they work? Thanks!

**Reply by spinicist** [reply]
Hello @marjoleinpiek,

Are you using the pre-compiled binaries for riesling or are you compiling yourself?

The binaries are a little bit out of date, I need to tag a new release ASAP.

**Comment by marjolein90** [comment]
Thanks for the update, I will give it another try!

Op do 10 jun. 2021 om 22:17 schreef Emil Ljungberg ***@***.***

**Comment by marjoleinpiek** [comment]
Dear Emil,


Thank you for the message and apologies for not responding earlier. The email got a bit lost.

I'm currently not using riesling, but I will keep it in mind.


Best,

Marjolein

________________________________
Van: Emil Ljungberg ***@***.***>
Verzonden: donderdag 17 juni 2021 11:49:11
Aan: spinicist/riesling
CC: Marjolein Piek; Comment
Onderwerp: Re: [spinicist/riesling] Recon_demo problem (#18)


In the riesling-examples repo there is a folder called rrsg_cg-sense. The file to download the data is called download_rrsg_data.sh. Can you have a look at those examples and see if they work? Thanks!

**Comment by marjoleinpiek** [comment]
Dear Tobias,


Thank you for the message and apologies for not responding earlier. The email got a bit lost.

I'm currently not using riesling, but I will keep it in mind.


Best,

Marjolein


________________________________
Van: Tobias Wood ***@***.***>
Verzonden: vrijdag 18 juni 2021 15:55:40
Aan: spinicist/riesling
CC: Marjolein Piek; Mention
Onderwerp: Re: [spinicist/riesling] Recon_demo problem (#18)


Hello @marjoleinpiek<https://github.com/marjoleinpiek>,

Are you using the pre-compiled binaries for riesling or are you compiling yourself?

The binaries are a little bit out of date, I need to tag a new release ASAP.
